trino/client.py

```diff
@@ -226,6 +227,66 @@ def __repr__(self):
             )
         )
 
+    def __process_rows(self, rows, columns):
```

Comment:
Should probably just be a private function instead of the double underscore of a Python hook.

Reply:
Fixed